<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="FirefoxTest"
           tests="4"
           failures="2"
           errors="0"
           skipped="0"
           time="163.372">
  <properties />
            <testcase name="linkAccountCancelTest" time="43.645" classname="automationFramework.TestCases.LinkAccountTest">
                <failure type="java.lang.AssertionError"
                  message="expected [$5.00] but found [$17.40]"
              >
<![CDATA[java.lang.AssertionError: expected [$5.00] but found [$17.40]
  org.testng.Assert.fail(Assert.java:94)
  org.testng.Assert.failNotEquals(Assert.java:494)
  org.testng.Assert.assertEquals(Assert.java:123)
  org.testng.Assert.assertEquals(Assert.java:176)
  org.testng.Assert.assertEquals(Assert.java:186)
  automationFramework.TestCases.LinkAccountTest.linkAccountCancelTest(LinkAccountTest.java:253)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
  org.testng.SuiteRunner.run(SuiteRunner.java:240)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
  org.testng.TestNG.run(TestNG.java:1031)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:177)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:105)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
  org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
  org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:103)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:74)
]]>
      </failure>

          </testcase>
            <testcase name="linkAccountAlreadyLinkedException" time="43.373" classname="automationFramework.TestCases.LinkAccountTest">
                <failure type="java.lang.AssertionError"
                  message="expected [Account Already Linked] but found [Link Account Failure]"
              >
<![CDATA[java.lang.AssertionError: expected [Account Already Linked] but found [Link Account Failure]
  org.testng.Assert.fail(Assert.java:94)
  org.testng.Assert.failNotEquals(Assert.java:494)
  org.testng.Assert.assertEquals(Assert.java:123)
  org.testng.Assert.assertEquals(Assert.java:176)
  org.testng.Assert.assertEquals(Assert.java:186)
  automationFramework.TestCases.LinkAccountTest.linkAccountAlreadyLinkedException(LinkAccountTest.java:121)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
  org.testng.SuiteRunner.run(SuiteRunner.java:240)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
  org.testng.TestNG.run(TestNG.java:1031)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:177)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:105)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
  org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
  org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:103)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:74)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="linkAccountCardExpirationMissingTest" time="41.400" classname="automationFramework.TestCases.LinkAccountTest" />
                <testcase name="linkAccountBankNumberMissingTest" time="34.954" classname="automationFramework.TestCases.LinkAccountTest" />
      
</testsuite>
